vendor/ember/shims.js

```diff
@@ -0,0 +1,247 @@
+        'default':  Ember.onLoad,
+        'run':      Ember.runLoadHooks
+      },
+      'ember-metal/set': {
+        'default':        Ember.set,
+        'setProperties':  Ember.setProperties,
+        'trySet':         Ember.trySet
+      },
+      'ember-metal/utils': {
+        'aliasMethod':  Ember.aliasMethod,
+        'assert':       Ember.assert,
+        'cacheFor':     Ember.cacheFor,
+        'copy':         Ember.copy
+      },
+      'ember-object': {
+        'default': Ember.Object
+      },
+      'ember-platform': {
+        'assign':         Ember.merge,
+        'create':         Ember.create,
+        'defineProperty': Ember.platform.defineProperty,
+        'hasAccessors':   Ember.platform.hasPropertyAccessors,
+        'keys':           Ember.keys
+      },
+      'ember-route': {
+        'default': Ember.Route
+      },
+      'ember-router': {
+        'default': Ember.Router
+      },
+      'ember-runloop': {
+        'default':      Ember.run,
+        'begin':        Ember.run.begin,
+        'bind':         Ember.run.bind,
+        'cancel':       Ember.run.cancel,
+        'debounce':     Ember.run.debounce,
+        'end':          Ember.run.end,
+        'join':         Ember.run.join,
+        'later':        Ember.run.later,
+        'next':         Ember.run.next,
+        'once':         Ember.run.once,
+        'schedule':     Ember.run.schedule,
+        'scheduleOnce': Ember.run.scheduleOnce,
+        'throttle':     Ember.run.throttle
+      },
+      'ember-service': {
+        'default': Ember.Service
+      },
+      'ember-service/inject': {
+        'default': Ember.inject.service
+      },
+      'ember-set/ordered': {
+        'default': Ember.OrderedSet
+      },
+      'ember-string': {
+        'camelize':     Ember.String.camelize,
+        'capitalize':   Ember.String.capitalize,
+        'classify':     Ember.String.classify,
+        'dasherize':    Ember.String.dasherize,
+        'decamelize':   Ember.String.decamelize,
+        'fmt':          Ember.String.fmt,
+        'htmlSafe':     Ember.String.htmlSafe,
+        'loc':          Ember.String.loc,
+        'underscore':   Ember.String.underscore,
+        'w':            Ember.String.w
+      },
+      'ember-utils': {
+        'isBlank':    Ember.isBlank,
+        'isEmpty':    Ember.isEmpty,
+        'isNone':     Ember.isNone,
+        'isPresent':  Ember.isPresent,
+        'tryInvoke':  Ember.tryInvoke,
+        'typeOf':     Ember.typeOf
+      }
+    };
+
+    // populate `ember/computed` named exports
+    shims['ember-computed'] = {
+      'default': Ember.computed
+    };
+    var computedMacros = [
+      "empty","notEmpty", "none", "not", "bool", "match", "equal", "gt", "gte",
+      "lt", "lte", "alias", "oneWay", "reads", "readOnly", "deprecatingAlias",
+      "and", "or", "collect", "sum", "min", "max", "map", "sort", "setDiff",
+      "mapBy", "filter", "filterBy", "uniq", "union", "intersect"
```

Comment:
Do we need to add `uniqBy` here as well, now that it's been added in https://github.com/emberjs/ember.js/pull/12875 ?